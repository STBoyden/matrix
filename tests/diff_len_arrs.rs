// Test fails when the sub arrays are different lenghts
use matrix_multiplication::matrix;

fn main() {
    let _x = matrix!([[1, 2, 3], [4, 5], [6], [7, 8, 9]]);
}
