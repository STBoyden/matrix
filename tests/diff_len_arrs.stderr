error: All of the arrays within the matrix must be the same length.
 --> $DIR/diff_len_arrs.rs:5:34
  |
5 |     let _x = matrix!([[1, 2, 3], [4, 5], [6], [7, 8, 9]]);
  |                                  ^^^^^^
